packages/react-aria-components/test/Autocomplete.test.tsx

```diff
@@ -244,6 +244,136 @@ describe('Autocomplete', () => {
     expect(options[1]).toHaveAttribute('data-focused');
     expect(options[1]).not.toHaveAttribute('data-focus-visible');
   });
+
+  it('should be able to tab inside a focus scope that contains', async () => {
+    const MyMenu = () => {
+      let {contains} = useFilter({sensitivity: 'base'});
+
+      return (
+        <DialogTrigger>
+          <Button aria-label="Menu">☰</Button>
+          <Popover>
+            <Dialog>
+              <Button>First</Button>
+              <Button>Second</Button>
+              <UNSTABLE_Autocomplete filter={contains}>
+                <TextField autoFocus aria-label="Search">
+                  <Input />
+                </TextField>
+                <Menu>
+                  <MenuItem>Open</MenuItem>
+                  <MenuItem>
+                    Rename…
+                  </MenuItem>
+                  <MenuItem>
+                    Duplicate
+                  </MenuItem>
+                </Menu>
+              </UNSTABLE_Autocomplete>
+            </Dialog>
+          </Popover>
+        </DialogTrigger>
+      );
+    };
+
+    function App() {
+      return (
+        <div>
+          <input />
+          <div>
+            <MyMenu />
+          </div>
+          <input />
+        </div>
+      );
+    }
+
+    let {getByRole} = render(<App />);
+    let trigger = getByRole('button', {name: 'Menu'});
+    await user.click(trigger);
+    let firstButton = getByRole('button', {name: 'First'});
+    let secondButton = getByRole('button', {name: 'Second'});
+    let input = getByRole('textbox');
+
+    expect(document.activeElement).toBe(input);
+
+    await user.tab();
+
+    expect(document.activeElement).toBe(firstButton);
+
+    await user.tab({shift: true});
+
+    expect(document.activeElement).toBe(input);
+
+    await user.tab({shift: true});
+
+    expect(document.activeElement).toBe(secondButton);
+  });
+
+  it('should be able to tab inside a focus scope that contains', async () => {
```

Comment:
why are these two tests called the same thing?

Reply:
copy pasta